### PR TITLE
update ui schema editor

### PR DESCRIPTION
**Pull request by AhmetTanakol**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -12,16 +12,36 @@ import {
...
   'type': 'MasterDetailLayout',
   'scope': '#'
 };
 
+export const filterPredicate = (data: Object) => {
+  return (property: Property): boolean => {
+    if (!_.isEmpty(modelMapping) &&
+      !_.isEmpty(modelMapping.mapping)) {
+      if (data[modelMapping.attribute]) {
+        return property.schema.id === modelMapping.mapping[_.toLower(data[modelMapping.attribute])];
```
Comment (edgarmueller) [inline src/index.tsx]: Do we still need to use `toLower`?

**Review by edgarmueller — DISMISSED** [review]
LGTM, see comment about usage of `toLower`

File: src/index.tsx (hunk 1 above)
Comment (AhmetTanakol) [reply src/index.tsx]: yes, for instance type name is `HorizontalLayout`, and in my mapping configuration file I set the corresponding id to `horizontallayout`. If you want, I can edit config file

Comment (edgarmueller) [reply src/index.tsx]: Yes, please do that as that seems clearer to me.

**Comment by AhmetTanakol** [comment]
updated

**Review by edgarmueller — APPROVED** [review]

**Pull request merged**